samples/cpp/arrow/readme.html

```diff
@@ -930,34 +930,73 @@ <h1 class="title">Intel® Data Analytics Acceleration Library Apache Arrow* Samp
+<p>You can use Intel® DAAL Apache Arrow* samples on Linux*, Windows* and MacOS* operating systems.
 For a list of Intel® DAAL hardware and software requirements, refer to release notes
 for the version of Intel® DAAL you are using. </p>
     <h3>Apache Arrow* implementations against which Intel® DAAL has been validated:</h3>
     <ul>
-        <li>Apache Arrow* 0.13.0</li>
+        <li>Apache Arrow* 0.15.0</li>
     </ul>
-    <p><strong>Note:</strong> Intel® DAAL is expected to work on other Apache Arrow* version
+    <p><strong>Note:</strong> Intel® DAAL is expected to work on higher Apache Arrow* versions
     as well. Let us know if you have any troubles with the distribution you are using.</p>
     <h2><a name="Preparation_before_build">Preparation Before Build and Run</a></h2>
 <h3> Apache Arrow* support</h3>
 <p> You can download and install the Apache Arrow* library from the <a href="https://github.com/apache/arrow" target=_blank">
 https://github.com/apache/arrow </a>web page.
-Make sure to set the environment variables <samp class="codeph">ARROWROOT</samp> and <samp class="codeph">ARROWCONFIG</samp>
+Before running the sample make sure to set the environment variables <samp class="codeph">ARROWROOT</samp> and <samp class="codeph">ARROWCONFIG</samp>
  with the correct path and build type (debug or release, see <a href="https://github.com/apache/arrow/blob/master/docs/source/developers/cpp.rst#building" target=_blank">
-https://github.com/apache/arrow/blob/master/docs/source/developers/cpp.rst#building </a>web page for details.) before running the sample.</p>
+https://github.com/apache/arrow/blob/master/docs/source/developers/cpp.rst#building </a>web page for details.). For Linux* and MacOS* only: set variables <samp class="codeph">BOOST_SYSTEM_LIBRARY_PATH</samp> with the correct path to <samp class="codeph">libboost_system.a</samp> and <samp class="codeph">BOOST_FILESYSTEM_LIBRARY_PATH</samp> with the correct path to <samp class="codeph">libboost_filesystem.a</samp>.</p>
 <h3> Setting Up the Build Environment&nbsp;</h3>
 <p> Before you build the sample, you must set certain environment variables that define the location of related libraries. Intel® DAAL includes the <span class="filepath">vars</span> scripts that you can run to set environment variables:</p>
     <ul id="GUID-2FEE5E18-4507-402C-806D-F3C422B0D085">
+        <li>On Windows*, you can find the <span class="filepath">vars.bat</span> batch file at <span class="filepath">&lt;install-dir&gt;\compilers_and_libraries_xxxx.x.xxx\windows\daal\bin\:</span><samp class="codeph"><br />
+            vars.bat intel64
+            </samp>
+        </li>
         <li>On Linux*, you can find the <span class="filepath">vars.sh</span> shell script at <span class="filepath">&lt;install-dir&gt;\compilers_and_libraries_xxxx.x.xxx\linux\daal\bin:</span><samp class="codeph"><br />
-            source vars.sh {ia32|intel64}
+            source vars.sh intel64
```

Comment:
Why to specify intel64, if it's the only option?

Reply:
Comment concerning support of x64 architectures only was added below.